packages/framework/package.json

```diff
@@ -1,6 +1,6 @@
 {
-    "name": "@ebenos/framework",
-    "version": "4.0.0-alpha.29",
+    "name": "@ebonydevcopy/framework",
```

Comment:
shouldn't these references be changed here?

Reply:
Yes this pull request is just to keep track of the changes in the copy package. I'll make a new one with the correct changes for review and merging OTD

Reply:
cool